bgpd/bgp_zebra.c

```diff
@@ -1221,7 +1221,6 @@ void bgp_zebra_announce(struct bgp_dest *dest, const struct prefix *p,
 	unsigned int valid_nh_count = 0;
 	int has_valid_label = 0;
```

Comment:
I know it's not related to this change, but probably can be fixed using the same pattern for `has_valid_label` as well.

Reply:
Yes. I think it's not related to this change, so It shouldn't be fixed in this PR I think. It's okay to create another PR?

Reply:
ofc

Reply:
I created another PR to resolve this issue, please check it.
https://github.com/FRRouting/frr/pull/9651